src/shared/components/name-value-editor/_name-value-editor.scss

```diff
@@ -0,0 +1,72 @@
+$drag-column-width: 28px;
```

Comment:
Don't name files starting with `_`. Should be named as the component its being used in.

Reply:
I copied it from ODC which had the `_`, removed it.